UVicMartletplace/martletplace
# Update recommender endpoint to return charity status

# Description
The recommender endpoint needs to return listings marked for charity using their charityID, so the front end can display if the listing is part of a charity or not.

Closes #506 

## How to Test

- Build the application
- Create an account/login
- Notice some recommended items should be labeled for charity with a banner across their image.

## Checklist
